backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/nodes/AbstractPackageNodeDescriptionProvider.java

```diff
@@ -172,13 +172,16 @@ private DropNodeTool createDropFromDiagramTool(IViewDiagramElementFinder cache)
     }
 
     private NodeTool createNodeTool(NodeDescription nodeDescription, EClass eClass) {
+        var callElementInitializerService = this.viewBuilderHelper.newChangeContext()
+                .expression("aql:self.elementInitializer()");
+
         var setValue = this.viewBuilderHelper.newSetValue()
```

Comment:
`elementInitializer` service should set the default name of the newly created element, so I think you can remove this  `var setValue = ...` section.

Reply:
Correct, done.